kubernetes/api_deployment.yaml

```diff
@@ -19,12 +19,8 @@ spec:
         env:
           - name: DATASTORE_PLUGIN
             value: dynamodb
-          - name: HOST
-            value: 0.0.0.0
-          - name: API_URI
-            value: http://a4677c9873c9611e6aa7102b92f75d5c-1135862614.us-west-2.elb.amazonaws.com
-          - name: DATA_DIR
-            value: /opt/screwdriver/data
+          - name: URI
+            value: http://api.screwdriver.cd
```

Comment:
Can you add a comment here (since we are pointing users from the guide to this file) that they should put their `LoadBalancer Ingress` here?